gammapy/cube/exposure.py

```diff
@@ -48,6 +49,59 @@ def make_exposure_cube(pointing,
     )
 
 
+def make_background_cube(pointing,
+                         obstime,
+                         bkg,
+                         ref_cube,
+                         offset_max,
+                         ):
+    """Calculate background predicted counts cube.
+
+    This function evaluates the background rate model on
+    a sky cube, and then multiplies with
```

Comment:
This sentence is not finished...and maybe add a little bit more detail on the predicted background counts computation. Maybe add a link to the gamma astro data formats specification and mention here, that the predicted counts are computed 0th order (assuming a constant spectrum across the energy bin).

Reply:
Thanks! I've added a better description. (and also fixed some copy & paste error further down in the docstring)